site/en/docs/extensions/mv3/migrating_to_service_workers/index.md

```diff
@@ -252,6 +252,20 @@ function buildCanvas(width, height) {
 For additional guidance on working with `OffscreenCanvas`, see [OffscreenCanvas—Speed up Your Canvas
 Operations with a Web Worker][18].
 
+### Offscreen Documents
+
+For extensions that need to unobtrusively use APIs that require DOM access without visually opening up a new window or tab, developers can now use the [offscreen API](/docs/extensions/reference/offscreen/). This API allows developers to open and close undisplayed documents packaged with the extension for DOM-related tasks without disrupting the user experience. Offscreen documents do not share APIs with the extension's service worker, and function as fully functional web pages for the extension to interact with.
```

Comment:
>offscreenAPI

Please do one of two things. If you intend to refer to the API by a formal name, capitalize 'Offscreen API'. if you intended to refer to the namespace make `offscreen` API fixed width.